python/ray/autoscaler/_private/constants.py

```diff
@@ -50,6 +50,13 @@ def env_integer(key, default):
     "AUTOSCALER_MAX_CONCURRENT_LAUNCHES", 10
 )
 
+# Default upscaling speed for the autoscaler. This specified how many nodes
+# to request at a time, where the desired number to upscale is
+#   upscaling_speed * current_num_nodes.
+# e.g. 1.0 means to request enough nodes to double
+# the cluster size in each round of requests.
+DEFAULT_UPSCALING_SPEED = 0.0
```

Comment:
0 means we never add new nodes?

Reply:
current (legacy) behaviour is always sacling up by 1 node if needed to scale up. so something like below for scaling up a specific node type:
`num_scale_up = min(1, scaling_speed * (num_allocated+num_requested))`

Reply:
Add this to the comment